### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by artemgavrilov**
https://jira.percona.com/browse/PMM-0

- [ ] Links to other linked pull requests.

### CODE REVIEW

**Comment by artemgavrilov** [comment]
PMM-0

**Comment by artemgavrilov** [comment]
PMM-1

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+https://jira.percona.com/browse/PMM-0
```
Comment (artemgavrilov) [inline .github/pull_request_template.md:1]: Autolinks are unavailable for Percona-Lab namespace, so I used the full link here.

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2338-78b0784
client docker - perconalab/pmm-client-fb:PR-2338-78b0784
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2338-78b0784.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2338-78b0784&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2338-78b0784.tar.gz

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/12226/

**Review by yurkovychv — APPROVED** [review]

**Pull request merged**